src/expr/core/src/window_function/call.rs

```diff
@@ -437,6 +561,21 @@ impl<T> FrameBound<T> {
     }
 }
 
+impl<T> FrameBound<T>
+where
+    T: Copy,
+{
+    fn reverse(self) -> FrameBound<T> {
+        match self {
+            UnboundedPreceding => UnboundedFollowing,
+            Preceding(offset) => Following(offset),
+            CurrentRow => CurrentRow,
+            Following(offset) => Preceding(offset),
+            UnboundedFollowing => UnboundedPreceding,
```

Comment:
It seems that `FrameBound<usize>` and `FrameBound<RangeFrameOffset>` are so different that they don't really need to be variants of same type. How about just saparating them as 2 types, such as `RowsFrameBound` and `RangeFrameBound`?

Reply:
IMO although they have many different methods and are used very differently, in their core they are the same concept, so I decided to introduce a generic type. Also I didn't want to repeat the 5 enum variants for two times (will be 3 times when we support `GROUPS` frame later).

Maybe we can `type RowsFrameBound = FrameBound<usize>;` to make it easier to read.

Reply:
Introducing type aliases sounds good